src/assets/images/LavaGlow.json

```diff
@@ -68,17 +68,17 @@
 		"frame": {"x":24,"y":0,"w":40,"h":16},
 		"rotated": false,
 		"trimmed": true,
-		"spriteSourceSize": {"x":0,"y":22,"w":40,"h":16},
+		"spriteSourceSize": {"x":0,"y":0,"w":40,"h":16},
 		"sourceSize": {"w":64,"h":64},
 		"pivot": {"x":0.5,"y":0.5}
 	}},
 	"meta": {
 		"app": "http://www.codeandweb.com/texturepacker",
 		"version": "1.0",
-		"image": "WaterAO.png",
+		"image": "LavaGlow.png",

```

Comment:
why this change?

Reply:
Bad copy-paste by me earlier 😄.  The `LavaGlow.json` atlas is for `LavaGlow.png` (but it's the same layout as `WaterAO.png`, so I copied & pasted when doing the original alignment).